### ISSUE TITLE
[QUESTION] Use custom dataset lables

### ISSUE THREAD

**Issue by Avedena** [stylegan-human/StyleGAN-Human#35]
How I can Prepare my Dataset with Labels to Train this Model?
I lable my data set as provided here in the [PyTorch Docs](https://www.tensorflow.org/datasets/catalog/celeb_a)

Here a Example frome my CSV:
"Image_Name""test.png", "Male":True, "Hair_Black":False, "Hair_Brown":True

The original StyleGAN Lib use only Integer Lables. How I can get the benefits in Learning from my Dataset?
Thanks

**Comment by yumingj** [comment]
Hi @Avedena ，

you can use some pre-defined integer labels to replace your original labels. For example, use 0 to denote "Female: True", use 1 to denote "Male: True".

Thanks!
